.github/workflows/first-actions.yml

```
@@ -1,6 +1,9 @@
 name: My First GitHub Actions
 
-on: [push]
+on:
+  pull request:
```

actionlint (1.7.7)

unknown Webhook event "pull request". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)